src/serverenvironment.cpp

```diff
@@ -1926,9 +1922,16 @@ u16 ServerEnvironment::addActiveObjectRaw(std::unique_ptr<ServerActiveObject> ob
 			errorstream<<"ServerEnvironment::addActiveObjectRaw(): "
 				<<"could not emerge block for storing id="<<object->getId()
 				<<" statically (pos="<<p<<")"<<std::endl;
```

Comment:
consider replacing the above with a call to `saveStaticToBlock`. it implements the same.

Reply:
It does not look so.

`saveStaticToBlock` does not call `block->m_static_objects.setActive`.
Only the error message is similar.

Reply:
Indeed you are right.